contrib/gitlab.tpl

```diff
@@ -1,10 +1,11 @@
 {{- /* Template based on https://docs.gitlab.com/ee/user/application_security/container_scanning/#reports-json-format */ -}}
 {
-  "version": "2.3",
+  "version": "14.0.6",
   "vulnerabilities": [
   {{- $t_first := true }}
   {{- range . }}
   {{- $target := .Target }}
+    {{ $image := $target | regexFind "[^\\s]+" }}
```

Comment:
```suggestion
    {{- $image := $target | regexFind "[^\\s]+" -}}
```

@knqyf263 , @thiago-gitlab this should help with the error in `TestClientServerWithFormat`

Reply:
Many thanks, @DmitriyLewen. I'll fix that today 🙇🏽